phys/module_sf_noahdrv.F

```diff
@@ -105,7 +105,7 @@ SUBROUTINE lsm(DZ8W,QV3D,P8W3D,T3D,TSK,                      &
                   dl_u_bep,sf_bep,vl_bep,sfcheadrt,INFXSRT, soldrain      &   !O multi-layer urban         
                  ,SDA_HFX, SDA_QFX, HFX_BOTH, QFX_BOTH, QNORM, fasdas     &   !fasdas
                  ,RC2,XLAI2                                               &
-                  )
+                 ,IRR_CHAN)
```

Comment:
@arjanna 
Arianna,
Again, keeping with conventions, your added code would be 
```
   ,IRR_CHAN       &
```
Insert you code prior to the closing ")".